drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/strategies/AbstractDMODataManagementStrategy.java

```diff
@@ -156,10 +156,12 @@ protected void loadSuperTypes(final List<String> dataObjectsTypes,
     public FactModelTree getFactModelTree(String factName, Map<String, String> superTypeMap, ModelField[] modelFields) {
         Map<String, FactModelTree.PropertyTypeName> simpleProperties = new HashMap<>();
         Map<String, List<String>> genericTypesMap = new HashMap<>();
-        String factPackageName = packageName;
         String fullFactClassName = getFQCNByFactName(factName);
+        String factPackageName = packageName;
+        String factClassName = fullFactClassName;
```

Comment:
we don't need this assignment here, null would be a better choice IMO.

Reply:
@jstastny-cz That is the default value for `factClassName` variable. If I put `null` here, I need to define an `else` statement in line 166, defining `factClassName` variable. I think semantically it's the same.

Reply:
Ah, I missed that, thanks.